docs/source/en/model_doc/owlvit.mdx

```diff
@@ -0,0 +1,88 @@
+specific language governing permissions and limitations under the License.
+-->
+
+# OwlViT
+
+## Overview
+
+The Owl-ViT model was proposed in [Simple Open-Vocabulary Object Detection with Vision Transformers](https://arxiv.org/pdf/2205.06230.pdf) by Matthias Minderer, Alexey Gritsenko, Austin Stone, Maxim Neumann, Dirk Weissenborn, Alexey Dosovitskiy, Aravindh Mahendran, Anurag Arnab, Mostafa Dehghani, Zhuoran Shen, Xiao Wang, Xiaohua Zhai, Thomas Kipf, and Neil Houlsby. Owl-ViT is an open-vocabulary object detection network trained on a variety of (image, text) pairs. It can be used to query an image with one or multiple text queries to search for and detect target objects described in text.
+
+The abstract from the paper is the following:
+
+*Combining simple architectures with large-scale pre-training has led to massive improvements in image classification. For object detection, pre-training and scaling approaches are less well established, especially in the long-tailed and open-vocabulary setting, where training data is relatively scarce. In this paper, we propose a strong recipe for transferring image-text models to open-vocabulary object detection. We use a standard Vision Transformer architecture with minimal modifications, contrastive image-text pre-training, and end-to-end detection fine-tuning. Our analysis of the scaling properties of this setup shows that increasing image-level pre-training and model size yield consistent improvements on the downstream detection task. We provide the adaptation strategies and regularizations needed to attain very strong performance on zero-shot text-conditioned and one-shot image-conditioned object detection. Code and models are available on GitHub.*
+
+## Usage
+
+OwlViT is a zero-shot text-conditioned object detection model. OwlViT uses CLIP as its multi-modal backbone, with a ViT like transformer to get visual features and a causal language model to get the text features. To use CLIP for detection, OwlViT removes the final token pooling layer of the vision model and attaches a lightweight classification and box head to each transformer output token. Open-vocabulary classification is enabled by replacing the fixed classification layer weights with the class-name embeddings obtained from the text model. The authors first train CLIP from scratch and fine-tune it end-to-end with the classification and box heads on standard detection datasets using a bipartite matching loss. One or multiple text queries per image can be used to perform zero-shot text-conditioned object detection. 
+
+The [`CLIPFeatureExtractor`] can be used to resize (or rescale) and normalize images for the model and the [`CLIPTokenizer`] is used to encode the text. The [`OwlViTProcessor`] wraps [`CLIPFeatureExtractor`] and [`CLIPTokenizer`] into a single instance to both encode the text and prepare the images. The following example shows how to perform object detection using [`OwlViTProcessor`] and [`OwlViTForObjectDetection`].
```

Comment:
```suggestion
The [`CLIPFeatureExtractor`] can be used to resize (or rescale) and normalize images for the model and the [`CLIPTokenizer`] is used to encode the text. The [`OwlViTProcessor`] wraps [`CLIPFeatureExtractor`] and [`CLIPTokenizer`] into a single instance to both encode the text and prepare the images. The following example shows how to perform object detection using [`OwlViTProcessor`] and [`OwlViTForObjectDetection`].
```
Seems like there's a new OwlViTFeatureExtractor defined, instead of using the existing CLIPFeatureExtractor?

Reply:
Yes, that's correct, I updated it